.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -9,28 +9,30 @@ assignees: ''
 
 <!--
 Please read through the [guidelines](https://github.com/postmanlabs/postman-app-support#guidelines-for-reporting-issues) before creating a new issue.
+The more information that you can provide when raising an issue, will give us the best possible chance to quickly get to the root of the problem.
```

Comment:
Can we make that sentence easier to read? 
> Provide as much information as you can as this will give us the best possible chance to quickly get to the root of the problem.

Reply:
Sure - Any suggestions?